stix/idl/processing/spectrogram/stx_science_data_lightcurve.pro

```diff
@@ -67,11 +71,14 @@
 ;                               added keyword to allow the user to specify the systematic uncertainty
 ;    05-Sep-2022 - ECMD (Graz), added rcr info to output structure
 ;                               suppress plotting when ospex object is created  
+;    15-Dec-2023 - AFB (FHNW), keyword time_range added, to extract only a sub-interval
+;    31-Jan-2023 - AFB (FHNW), it is now possible to pass multiple FITS files in fits_path and the output
```

Comment:
maybe update the doc string for the `fits_path` to indicate this change?

Reply:
The same for `fits_path`